Exception running the RESP/API Docker container

I've been having issues trying to run the rest-api container as described in the documentation. I've been able to run the odinson-extra container successfully.  Running on MacOS BigSur.

running sample command (with the volume mapping modified to point. to the directory with the generated index)

```
docker run \
  --name="odinson-rest-api" \
  -it \
  --rm \
  -e "HOME=/app" \
  -p "0.0.0.0:9001:9000" \
  -v "/path/to/data/odinson:/app/data/odinson" \
  "lumai/odinson-rest-api:latest"
```

This  problem is present in the output log (multiple apparently similar Exceptions are indicated):

```
Oops, cannot start the server.
com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Error injecting constructor, com.typesafe.config.ConfigException$Missing: reference.conf @ jar:file:/opt/docker/lib/ai.lum.odinson-core-0.4.1-SNAPSHOT.jar!/reference.conf: 75: No configuration setting found for key 'odinson.index.parentDocFieldFileName'
  at controllers.OdinsonController.<init>(OdinsonController.scala:42)
  at controllers.OdinsonController.class(OdinsonController.scala:41)
  while locating controllers.OdinsonController
    for the 3rd parameter of router.Routes.<init>(Routes.scala:37)
  while locating router.Routes
  while locating play.api.inject.RoutesProvider
  while locating play.api.routing.Router
    for the 3rd parameter of play.api.http.JavaCompatibleHttpRequestHandler.<init>(HttpRequestHandler.scala:298)
  while locating play.api.http.JavaCompatibleHttpRequestHandler
  while locating play.api.http.HttpRequestHandler
    for the 6th parameter of play.api.DefaultApplication.<init>(Application.scala:176)
  at play.api.DefaultApplication.class(Application.scala:175)
  while locating play.api.DefaultApplication
  while locating play.api.Application
Caused by: com.typesafe.config.ConfigException$Missing: reference.conf @ jar:file:/opt/docker/lib/ai.lum.odinson-core-0.4.1-SNAPSHOT.jar!/reference.conf: 75: No configuration setting found for key 'odinson.index.parentDocFieldFileName'
	at com.typesafe.config.impl.SimpleConfig.findKeyOrNull(SimpleConfig.java:156)
	at com.typesafe.config.impl.SimpleConfig.findOrNull(SimpleConfig.java:174)
	at com.typesafe.config.impl.SimpleConfig.findOrNull(SimpleConfig.java:180)
	at com.typesafe.config.impl.SimpleConfig.findOrNull(SimpleConfig.java:180)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:188)
	at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:193)
	at com.typesafe.config.impl.SimpleConfig.getString(SimpleConfig.java:250)
	at ai.lum.common.ConfigUtils$StringConfigFieldReader$.read(ConfigUtils.scala:88)
	at ai.lum.common.ConfigUtils$StringConfigFieldReader$.read(ConfigUtils.scala:86)
	at ai.lum.common.ConfigUtils$LumAICommonConfigWrapper$.apply$extension(ConfigUtils.scala:36)
	at controllers.OdinsonController.<init>(OdinsonController.scala:60)
	at controllers.OdinsonController$$FastClassByGuice$$db30fd5d.newInstance(<generated>)
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1050)
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1086)
	at play.api.inject.guice.GuiceInjector.instanceOf(GuiceInjectorBuilder.scala:435)
	at play.api.inject.ContextClassLoaderInjector.$anonfun$instanceOf$3(Injector.scala:117)
	at play.api.inject.ContextClassLoaderInjector.withContext(Injector.scala:125)
	at play.api.inject.ContextClassLoaderInjector.instanceOf(Injector.scala:117)
	at play.api.inject.RoutesProvider.$anonfun$get$2(BuiltinModule.scala:119)
	at scala.Option.fold(Option.scala:251)
	at play.api.inject.RoutesProvider.get$lzycompute(BuiltinModule.scala:119)
	at play.api.inject.RoutesProvider.get(BuiltinModule.scala:114)
	at play.api.inject.RoutesProvider.get(BuiltinModule.scala:108)
	at com.google.inject.internal.ProviderInternalFactory.provision(ProviderInternalFactory.java:85)
	at com.google.inject.internal.BoundProviderFactory.provision(BoundProviderFactory.java:77)
	at com.google.inject.internal.ProviderInternalFactory.circularGet(ProviderInternalFactory.java:59)
	at com.google.inject.internal.BoundProviderFactory.get(BoundProviderFactory.java:61)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:62)
	at com.google.inject.internal.SingleParameterInjector.inject(SingleParameterInjector.java:42)
	at com.google.inject.internal.SingleParameterInjector.getAll(SingleParameterInjector.java:65)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:113)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39)
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:62)
	at com.google.inject.internal.InternalInjectorCreator.loadEagerSingletons(InternalInjectorCreator.java:211)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:182)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:109)
	at com.google.inject.Guice.createInjector(Guice.java:87)
	at com.google.inject.Guice.createInjector(Guice.java:78)
	at play.api.inject.guice.GuiceBuilder.injector(GuiceInjectorBuilder.scala:200)
	at play.api.inject.guice.GuiceApplicationBuilder.build(GuiceApplicationBuilder.scala:155)
	at play.api.inject.guice.GuiceApplicationLoader.load(GuiceApplicationLoader.scala:21)
	at play.core.server.ProdServerStart$.start(ProdServerStart.scala:53)
	at play.core.server.ProdServerStart$.main(ProdServerStart.scala:29)
	at play.core.server.ProdServerStart.main(ProdServerStart.scala)

```

--- a/backend/conf/application.conf
+++ b/backend/conf/application.conf
@@ -1,5 +1,7 @@
-# Configuration for Play's AkkaHttpServer
+# Configurations for ensuring the backend can properly use the index
+odinson.index.parentDocFieldFileName = fileName
 
+# Configuration for Play's AkkaHttpServer
 play {
 
   http {

